### PR TITLE
Kindernachmittag, Language Polishing

### PR DESCRIPTION
**Pull request by Glottotopia**
I added the Kindernachmittag after the Open Monday info

I updated member figures and relative year figures "....4 years ago ..." --> 2018

I did some language polishing for English and German

I removed the embroidery machine and added the knitting machine.

### CODE REVIEW

File: content/en/support.md update member figures, some language polishing [hunk 1]
```diff
@@ -5,9 +5,9 @@ draft: false
...
-We currently have 130 members who support us. We organise Workshops for adults over a wide variety of topics that cover hacking and making. We cooperate with CoderDojo Germany e.V. and Jugend Hackt where young people can learn programming and other skills.
+Our hack- and makespace, xHain, has been around since 2018. xHain is an open workshop and community for everyone interested in hacking and making, and has become an important meeting place for people in Berlin.  
+xHain is a non-profit organization that enables anyone to build projects, provides space for many important contemporary areas, such as discussing digital politics. Over the years, it has become an essential place for organising youth activities.
+We currently have 200 members who support us. We organise Workshops for adults over a wide variety of topics that cover hacking and making. We cooperate with CoderDojo Germany e.V. and Jugend Hackt, where young people can learn programming and other skills.
```
Comment (jdwillemse) [inline content/en/support.md update member figures, some language polishing:10]: The single line breaks are misleading as they are not rendered as paragraphs. I suggest removing them or making actual paragraph breaks if that is the intention.

File: content/en/support.md update member figures, some language polishing [hunk 2]
```diff
@@ -5,9 +5,9 @@ draft: false
...
-xHain is a non-profit organization that enables anyone to build projects, provides space for many important contemporary areas, such as discussing digital policy, and through the years has become an essential place for organising youth work.
-We currently have 130 members who support us. We organise Workshops for adults over a wide variety of topics that cover hacking and making. We cooperate with CoderDojo Germany e.V. and Jugend Hackt where young people can learn programming and other skills.
+Our hack- and makespace, xHain, has been around since 2018. xHain is an open workshop and community for everyone interested in hacking and making, and has become an important meeting place for people in Berlin.  
+xHain is a non-profit organization that enables anyone to build projects, provides space for many important contemporary areas, such as discussing digital politics. Over the years, it has become an essential place for organising youth activities.
```
Comment (jdwillemse) [inline content/en/support.md update member figures, some language polishing:9]: A suggestion for simplification:

xHain is a non-profit organization that provides a space for anyone to build projects, learn new skills, and discuss important topics like digital politics. Over the years it has also become a hub for youth activities.

File: content/en/support.md update member figures, some language polishing (hunk 1 above)
Comment (jdwillemse) [reply content/en/support.md update member figures, some language polishing:10]: Copy suggestion:

We have 200 members who support us. We organize workshops for adults on a wide variety of topics, including hacking and making. We partner with CoderDojo Germany e.V. and Jugend Hackt, where young people can learn programming and other skills.

File: content/en/_index.md [hunk 3]
```diff
@@ -5,13 +5,13 @@ draft: false
...
 ## Welcome to xHain
-**Hack- & Makespace in the heart of Friedrichshain, tinkering room for people interested in technology + space for net politics.**
+**Hack- & Makespace in the heart of Friedrichshain, tinkering room for people interested in technology + space for digital politics.**
 
 <br clear="all">
 
 ## Open monday nights
 
-Come by, take a look and join in! **Every Monday from 6pm until late at night in Grünberger Str. 16** in Berlin.
+Come by, take a look, and join in! **Every Monday from 6pm until late at night in Grünberger Str. 16** in Berlin. For children, we offer one Open Kids Sunday a month (1pm-6pm, typically towards the end of the month).
```
Comment (jdwillemse) [inline content/en/_index.md:14]: Suggestion:

We have an open event for kids every month. It is typically held towards the end of the month and runs from 1 – 6 p.m.

**Review by jdwillemse — COMMENTED** [review]
I have a few copy suggestions

**Review by pantierra — APPROVED** [review]

**Comment by pantierra** [comment]
> I have a few copy suggestions

Thanks a lot @jdwillemse! I just applied your suggestions after merging this PR in.

**Comment by pantierra** [comment]
Thanks a lot for the language review @Glottotopia!

**Pull request merged**